Unable to deploy function

Continuing on past trying to run the function locally i try deploying the function using - 
mvn azure-functions:deploy

And get this output - 
[INFO] Scanning for projects...
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/maven-metadata.xml
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-metadata.xml (14 kB at 11 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/maven-metadata.xml (20 kB at 16 kB/s)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.083 s
[INFO] Finished at: 2019-07-02T13:59:12+12:00
[INFO] ------------------------------------------------------------------------
[ERROR] No plugin found for prefix 'azure-functions' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (C:\Users\ryanc\.m2\repository), central (https://repo.maven.apache.org/maven2)]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e656bde5-b5d8-8d80-b7b3-ab430e57613e
* Version Independent ID: 329eb12f-7302-4ecf-c1bf-a50ba3f34bdf
* Content: [Create your first function in Azure with Java and Maven](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-java-maven#feedback)
* Content Source: [articles/azure-functions/functions-create-first-java-maven.md](https://github.com/Microsoft/azure-docs/blob/master/articles/azure-functions/functions-create-first-java-maven.md)
* Service: **azure-functions**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw, glenga**

--- a/articles/azure-functions/functions-create-first-java-maven.md
+++ b/articles/azure-functions/functions-create-first-java-maven.md
@@ -110,7 +110,7 @@ public class Function {
 
 ## Run the function locally
 
-Change directory to the newly created project folder and build and run the function with Maven:
+Change directory to the newly created project folder (the one containing your host.json and pom.xml files) and build and run the function with Maven:
 
 ```
 cd fabrikam-function

